content/en/logs/log_collection/android.md

```diff
@@ -498,13 +498,24 @@ By default, the following attributes are added to all logs sent by a logger:
 Use the `addAttribute("<ATTRIBUTE_KEY>", "<ATTRIBUTE_VALUE>")` function to add a custom attribute to all logs sent by a specific logger:
 
 ```kotlin
-// This adds an attribute "version_code" with an integer value
+// This adds an attribute "version_code" with an integer value in this logger instance
 logger.addAttribute("version_code", BuildConfig.VERSION_CODE)
 
-// This adds an attribute "version_name" with a String value
+// This adds an attribute "version_name" with a String value in this logger instance
 logger.addAttribute("version_name", BuildConfig.VERSION_NAME)
 ```
 
+Attributes can be added globally across all Logs instances (e.g., service name, environment) by using:
+
+```kotlin
+
+// This adds an attribute "version_code" with an integer value in all Logs instances.
```

Comment:
```suggestion
```kotlin
// This adds an attribute "version_code" with an integer value in all Logs instances.
```